lib/render/mayaToHydra/defaultLightDelegate.h

```diff
@@ -0,0 +1,59 @@
+//
+// Copyright 2019 Luma Pictures
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#ifndef __MTOH_DEFAULT_LIGHT_DELEGATE_H__
+#define __MTOH_DEFAULT_LIGHT_DELEGATE_H__
+
+#include <pxr/pxr.h>
+
+#include <pxr/imaging/glf/simpleLight.h>
+#include <pxr/imaging/hd/renderIndex.h>
+
+#include <pxr/usd/sdf/path.h>
+
+#include <pxr/usd/usd/stage.h>
+#include <pxr/usdImaging/usdImaging/delegate.h>
+
+#include <memory>
```

Comment:
Is it required?

Reply:
Nope, good catch - fixed in https://github.com/Autodesk/maya-usd/pull/231/commits/d1b349f8ff22401c38f9ae7986c78306b58e4144